.gitignore

```diff
@@ -7,3 +7,4 @@ release
 *.save
 build*
 *.autosave
+**/.qmake.stash

```

Comment:
Why not just `.qmake.stash`

Reply:
> Why not just `.qmake.stash`

It was in the all the submodule directories, this should just remove `.qmake.stash` from all the submodules.